.github/workflows/reusable_testing.yml

```diff
@@ -0,0 +1,46 @@
+name: reusable_testing
+
+on:
+  workflow_call:
+    inputs:
+      artifact_name:
+        description: The name of the tarantool build artifact
+        default: ubuntu-focal
+        required: false
+        type: string
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Clone the memcached module
+        uses: actions/checkout@v2
+        with:
+          repository: ${{ github.repository_owner }}/memcached
+          # Enable recursive submodules checkout as test-run git module is used
+          # for running tests.
+          submodules: recursive
+
+      - name: Download the tarantool build artifact
+        uses: actions/download-artifact@v2
+        with:
+          name: ${{ inputs.artifact_name }}
+
+      - name: Install tarantool
+        # Now we're lucky: all dependencies are already installed. Check package
+        # dependencies when migrating to other OS version.
+        run: sudo dpkg -i tarantool*.deb
```

Comment:
The same question as in https://github.com/tarantool/expirationd/pull/88#discussion_r731594884.
Could you use `run: sudo dpkg -i tarantool_*.deb tarantool-common_*.deb tarantool-dev_*.deb` instead?

Reply:
Yes, sure. Fixed.